src/renderer/base/RenderSettings.cpp

```diff
@@ -252,6 +252,8 @@ std::pair<COLORREF, COLORREF> RenderSettings::GetAttributeColors(const TextAttri
             fg = bg;
         }
 
+        fg = GetRenderMode(Mode::AlwaysDistinguishableColors) ? ColorFix::GetPerceivableColor(fg, bg) : fg;

```

Comment:
consider: this will break concealed text ;)

Reply:
Really should've caught that. Fixed, thank you!